sodetlib/operations/bias_steps.py

```diff
@@ -467,7 +470,8 @@ def _load_am(self, arc=None, fix_timestamps=True):
             if arc:
                 self.am = arc.load_data(self.start, self.stop)
             else:
-                self.am = sdl.load_session(self.meta['stream_id'], self.sid)
+                self.am = sdl.load_session(self.meta['stream_id'], self.sid,
```

Comment:
It doesn't look like `load_session` can currently take a `base_dir` argument. Also, out of curiosity, what issue is this PR addressing?

Reply:
https://github.com/simonsobs/sodetlib/blob/4aafb94bb36cac85b754b0a1e28575d33e1b6243/sodetlib/stream.py#L71-L72

Reply:
Interesting...I was looking at `load_session` here: https://github.com/simonsobs/sodetlib/blob/4aafb94bb36cac85b754b0a1e28575d33e1b6243/sodetlib/legacy/smurf_funcs/smurf_ops.py#L422 not sure why we have a duplicate.

Reply:
Ahh it's in "legacy" just searched in github and it gave me that one. Ok, this all makes sense we should just accept this PR probably but I'll wait an hr or so for @jlashner to have a chance to take a look.

Reply:
I linked Issue #214 , which has a slightly different title, but the key aspect there being there was no base_dir kwarg being passed to the load_session call in the bias step analysis.

This problem arises when trying to analyze bias steps without an G3tSMuRF archive on a machine other than the smurf server. For example, my timestreams on simons1 are not located at /data/so/timestreams.